README.md

```diff
@@ -1,10 +1,26 @@
-# 👷 Durable Objects Counter template
+# ♠️♥️ Pokerface ♣️♦️
+### A planning poker bot for Slack, on a Cloudflare Worker with Durable Objects
 
-## Note: You must use [wrangler](https://developers.cloudflare.com/workers/cli-wrangler/install-update) 1.19.3 or newer to use this template.
+## Getting started
 
-## Please read the [Durable Object documentation](https://developers.cloudflare.com/workers/learning/using-durable-objects) before using this template.
+- Clone https://github.com/autotelic/pokerface
+- Checkout `feat/avatars`
+- Run `miniflare -w -d -b [SLACK_TOKEN]`
+- Run `ngrok http --region=us --hostname=pokerface.ngrok.io 8787`
 
-A template for kick starting a Cloudflare Workers project using:
+## How to use
+
+- Create a group DM in Slack with the planning poker group
+- Invite PokerfaceDev to the DM
```

Comment:
Inviting PokerfaceDev after creating the group DM will actually create a new group DM
```suggestion
- Create a group DM in Slack with the planning poker group & PokerfaceDev
```